src/textual/_types.py

```diff
@@ -1,13 +1,12 @@
-from typing import Awaitable, Callable, List, TYPE_CHECKING, Union
+from typing import Callable, List, Union
+from typing import Awaitable, TYPE_CHECKING
```

Comment:
Here you import `Awaitable` from `typing` but you've imported it from `typing_extensions` above. (I left a comment there.)